src/projects/consultations.njk

```diff
@@ -1,15 +1,15 @@
 ---
-title: Consultations
+title: Consult
 layout: base.njk
 templateEngineOverride: njk
-ogTitle: i.AI Consultation Analyser
+ogTitle: Consult
 ogImage: https://ai.gov.uk/img/consultation4.png
 ogUrl: https://ai.gov.uk/projects/consultations
```

Comment:
Do we want to change the url as well?

Reply:
I was weary about doing that for SEO / if anyone has bookmarked it. I may be overthinking it and happy to change it if you feel strongly about it?